### PR TITLE
Enable autoretrieve

### PR DESCRIPTION
**Pull request by gmelodie**
Fix flags and logging to get the autoretrieve plugin to work. After this is merged we can start AR up again :)

### CODE REVIEW

File: autoretrieve/autoretrieve.go [hunk 1]
```diff
@@ -134,7 +137,7 @@ func NewAutoretrieveEngine(ctx context.Context, cfg *config.Estuary, db *gorm.DB
 			return nil, fmt.Errorf("no new CIDs to announce")
 		}
 
-		log.Infof("announcing %d new CIDs", len(newCids))
+		arLog.Infof("announcing %d new CIDs", len(newCids))
```
Comment (en0ma) [inline autoretrieve/autoretrieve.go:140]: please make this debugf

File: autoretrieve/autoretrieve.go [hunk 2]
```diff
@@ -179,7 +182,7 @@ func (arEng *AutoretrieveEngine) announceForAR(ar Autoretrieve) error {
 		return fmt.Errorf("no retrieval addresses for autoretrieve %s, skipping", ar.Handle)
 	}
 
-	log.Infof("sending announcement to %s", ar.Handle)
+	arLog.Infof("sending announcement to %s", ar.Handle)
```
Comment (en0ma) [inline autoretrieve/autoretrieve.go:185]: debugf

File: autoretrieve/autoretrieve.go [hunk 3]
```diff
@@ -190,7 +193,7 @@ func (arEng *AutoretrieveEngine) announceForAR(ar Autoretrieve) error {
 		return fmt.Errorf("unable to update advertisement time on database: %s", err)
 	}
 
-	log.Infof("announced new CIDs: %s", adCid)
+	arLog.Infof("announced new CIDs: %s", adCid)
```
Comment (en0ma) [inline autoretrieve/autoretrieve.go:196]: debugf

File: autoretrieve/autoretrieve.go [hunk 4]
```diff
@@ -223,11 +228,11 @@ func (arEng *AutoretrieveEngine) Run() {
 			}
 		}
 
-		log.Infof("announcing new CIDs to %d autoretrieve servers", len(autoretrieves))
+		arLog.Infof("announcing new CIDs to %d autoretrieve servers", len(autoretrieves))
```
Comment (en0ma) [inline autoretrieve/autoretrieve.go:231]: debugf

**Review by en0ma — DISMISSED** [review]
LGTM

**Review by en0ma — APPROVED** [review]
LGTM

**Review by alvin-reyes — APPROVED** [review]
LGTM

**Pull request merged**